packages/expo-cli/src/commands/build-native/index.ts

```diff
@@ -0,0 +1,35 @@
+import { UserManager, User } from '@expo/xdl';
+import { Command } from 'commander';
+
+import log from '../../log';
+import Builder from './Builder';
+import { Options } from './prepare';
+import { printBuildTable } from './utils'
+
+async function buildAction(projectDir: string, options: Options) {
+  const user: User = await UserManager.ensureLoggedInAsync();
+  const builder = new Builder(user);
+  const buildArtifactUrl = await builder.buildProject(projectDir, options);
+  log(`Artifact url: ${buildArtifactUrl}`);
+}
+
+async function statusAction() {
+  const user: User = await UserManager.ensureLoggedInAsync();
+  const builder = new Builder(user);
+  const result = await builder.getLatestBuilds();
+  printBuildTable(result.builds);
+}
+
+export default function (program: Command) {
+  program
+    .command('build-native [project-dir]')
+    .description('Build an app binary for your project, signed and ready for submission to the Google Play Store / App Store.')
+    .option('-p --platform <platform>', 'Platform: [android|ios]', /^(android|ios)$/i)
```

Comment:
currently if platform is missing command is failing after uploading to s3, make platform required option